pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BaseBrokerStarter.java

```diff
@@ -232,26 +232,27 @@ public void start()
     boolean caseInsensitive =
         _brokerConf.getProperty(Helix.ENABLE_CASE_INSENSITIVE_KEY, false) || _brokerConf.getProperty(
             Helix.DEPRECATED_ENABLE_CASE_INSENSITIVE_KEY, false);
-    TableCache tableCache = new TableCache(_propertyStore, caseInsensitive);
+    DefaultPinotConfigProvider defaultPinotConfigProvider =
```

Comment:
Probably use `PinotConfigProvider` as the reference here since we've already declared all the essential methods to the interface? Also change the variable name to `pinotConfigProvider`.

Reply:
i tried to, but there's still `isCaseSensitive` and `getActualTableName` methods used from the impl. 
I could just skip this PR and figure out what to do with those and then re-open, or I could just get the renaming in and then figure those out in a follow-up (figuring that might take some tie since I don't have the exact context on why thay are there in the first place)